### PR TITLE
feat: Add tabs childs redefinition

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
#### Screenshot or Gif

https://user-images.githubusercontent.com/20288327/126259860-074b525b-653b-446c-8ea4-9f889aab33d5.mp4

#### Other relevant information
* Your OS: Linux Mint 19.1 Cinnamon x64. 
* Browser: Mozilla Firefox 88.0.1.
* Node.js version: 14.17.0.
* NPM version: 7.19.0.
* adempiere-vue version: 4.3.1.

### CODE REVIEW

**Comment by EdwinBetanc0urt** [comment]
https://user-images.githubusercontent.com/20288327/126430447-698b1209-91a4-46dd-b32c-c97e869180d0.mp4

File: src/views/ADempiere/Window/index.vue [hunk 1]
```diff
@@ -32,23 +32,25 @@
...
-    v-loading="!isLoaded"
-    :element-loading-text="$t('notifications.loading')"
-    element-loading-background="rgba(255, 255, 255, 0.8)"
-    class="view-loading"
   />
 </template>
 
 <script>
 import { defineComponent, computed, ref } from '@vue/composition-api'
+
```
Comment (yamelsenih) [inline src/views/ADempiere/Window/index.vue:49]: Where is the MultiTab implementation?

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/views/ADempiere/Window/index.vue (hunk 1 above)
Comment (yamelsenih) [reply src/views/ADempiere/Window/index.vue:49]: ?

Comment (EdwinBetanc0urt) [reply src/views/ADempiere/Window/index.vue:49]: multitab implement

**Pull request merged**